### PR TITLE
Do not submit: Bumping apko to test perf

### PR DESCRIPTION
**Pull request by jonjohnsonjr**

### CODE REVIEW

**Comment by jonjohnsonjr** [comment]
```
No space left on device : '/home/runner/runners/2.305.0/_diag/pages/68a6e24e-a7bd-411f-8147-5eb2fbd10e4e_9af15312-ed5b-5608-26ca-4ae451e4ad7c_3.log'
```

lol

**Comment by jonjohnsonjr** [comment]
```
202023-07-08T23:28:13.1511101Z ##[error]No space left on device : '/home/runner/runners/2.305.0/_diag/pages/eef52252-d2d5-4f30-907f-14c281e70ac8_9af15312-ed5b-5608-26ca-4ae451e4ad7c_3.log'
```

This time in only 14 minutes! Progress!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-go@v4
       with:
-        go-version: '1.18'
+        go-version: '1.20'
```
Comment (mattmoor) [inline .github/workflows/test.yml:15]: we should check the release script too, just to be sure there's nothing else.

**Comment by mattmoor** [comment]
@jonjohnsonjr Can you build the failing test image before/after and figure out why the digest is changing, and if it's reasonable?

**Comment by jonjohnsonjr** [comment]
> Can you build the failing test image before/after and figure out why the digest is changing, and if it's reasonable?

We would expect it to change due to https://github.com/chainguard-dev/apko/pull/782

**Pull request closed without merging**